src/Components/SmartComponents/SystemsPage/SystemsPage.js

```diff
@@ -25,14 +25,14 @@ import useDeepCompareEffect from 'use-deep-compare-effect';
 import * as APIHelper from '../../../Helpers/APIHelper';
 
 const createColumns = (defaultColumns, hasPatchAccess) => {
-    let [nameColumn, restColumns] = SYSTEMS_HEADER;
+    let [nameColumn, ...restColumns] = SYSTEMS_HEADER;
     let lastSeenColumn = defaultColumns.filter(({ key }) => key === 'updated');
     let tagsColumn = defaultColumns.filter(({ key }) => key === 'tags');
 
-    tagsColumn = { ...tagsColumn[0], props: { width: 20, isStatic: true } };
+    tagsColumn = { ...tagsColumn[0], props: { width: 10, isStatic: true } };
     lastSeenColumn = { ...lastSeenColumn[0], cellTransforms: [nowrap], props: { width: 20 } };
 
-    let mergedColumns = [nameColumn, tagsColumn, restColumns, lastSeenColumn];
+    let mergedColumns = [nameColumn, tagsColumn, ...restColumns, lastSeenColumn];
 
     if (hasPatchAccess) {
```

Comment:
On systems page it's very unlikely to store `advisory` information. The advisory is linked to the CVE. Thus, the following code will never run

Reply:
@katerinapat So I can safely remove all code relating to advisories and patch access from systems page?